ci/docker/runtime_functions.sh

```diff
@@ -1922,6 +1922,27 @@ build_docs_beta() {
     popd
 }
 
+push_docs() {
+    folder_name=$1
+    set -ex
+    pip3 install --user awscli
+    export PATH=~/.local/bin:$PATH
+    pushd docs/_build
+    wget https://mxnet-website-static-artifacts.s3.us-east-2.amazonaws.com/versions.zip && unzip versions.zip && rm versions.zip
+    mkdir $folder_name && tar -xzf full_website.tgz -C $folder_name --strip-components 1
+    # check if folder_name already exists in versions
+    pushd versions
+    if [ -d "$folder_name" ]; then
+        echo "Folder $folder_name already exists in versions. Please double check the FOLDER_NAME variable in Jenkens pipeline"
+        exit 1
+    fi
+    popd
+    mv $folder_name versions
+    zip -r9 versions.zip versions/.
+    aws s3 cp versions.zip s3://mxnet-website-static-artifacts --acl bucket-owner-full-control
```

Comment:
Yes, It should be accessible since I passed the `bucket-owner-full-control` to the s3 bucket. The uploaded file should have the same permissions as other files in the bucket.

Reply:
We do not want to change the s3 bucket accessibility permissions with this PR. @ys2843 can you please confirm that `public-read` is the right permission for the bucket?